.code-samples.meilisearch.yaml

```diff
@@ -756,26 +780,26 @@ geosearch_guide_sort_usage_2: |-
     .await
     .unwrap();
 get_one_key_1: |-
-  let key = client.get_key("d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4").await.unwrap();
+  let key = client.get_key("6062abda-a5aa-4414-ac91-ecd7944c0f8d").await.unwrap();
 get_all_keys_1: |-
-  let keys = client.get_keys().await.unwrap();
+  let mut query = KeysQuery::new().with_limit(3).execute(&client).await.unwrap();
```

Comment:
Some of the examples are multiline and some are not, I think it could be nice to follow a standard :)

Reply:
If I bring consistency now I have to do it on every sample even the ones that are not linked to this PR. 
Maybe an issue that request that change ?

Reply:
Perfect for me! You can open the issue :)